### ISSUE TITLE
Hivelord stabilizers don't actually stabilize hivelord remains.

### ISSUE THREAD

**Issue by KingPhilipIII** [ParadiseSS13/Paradise#8354]
**Problem Description**:
See title.

**What did you expect to happen**: 
For my hivelord remains to save my ass, considering I saved it just for that.

**What happened instead**:
My remains were inert and useless.

**Why is this bad/What are the consequences:**
Useless item in the vendor, obviously broken function.

**Steps to reproduce the problem**:
1. Kill hivelord.
2.Buy stabilizer.
3.Use stabilizer on remains.
4.Wait for a while until you need the remains.
5. Die horribly when your specially prepared rejuv fails to save you.
6. Cry and salt in deadchat.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Nein.

**Comment by FalseIncarnate** [comment]
Just to make sure, the remains hadn't already gone inert, right?

**Comment by KingPhilipIII** [comment]
@FalseIncarnate 
Yes, the remains cannot be stabilized if they've already gone inert anyways.

**Comment by FalseIncarnate** [comment]
Just double checking, I try to rule out the most obvious troubleshooting steps first before delving too deeply into the issue. You'd be amazed how frequently people miss obvious steps, clear error messages, or even explicit instructions on how to do things that are clearly right in front of them the entire time.

**Comment by KingPhilipIII** [comment]
Yea I get it. Was just saying that there was even a mechanical reason that couldn't be the case.

**Comment by FalseIncarnate** [comment]
Actually, you can stabilize an inert core, it never bothers checking if the core is inert when you try to stabilize it. This is likely unintentional, but worth noting is possible.

**Comment by KingPhilipIII** [comment]
Oh it doesn't? Well, the more you know. In any case I'm pretty sure it wasn't inert because I kept checking it's description up until I preserved it to see if it was still good.
